hack/make-rules/test-integration.sh

```diff
@@ -29,9 +29,14 @@ export KUBE_CACHE_MUTATION_DETECTOR
 KUBE_PANIC_WATCH_DECODE_ERROR="${KUBE_PANIC_WATCH_DECODE_ERROR:-true}"
 export KUBE_PANIC_WATCH_DECODE_ERROR
 
+KUBE_INTEGRATION_TEST_MAX_CONCURRENCY=${KUBE_INTEGRATION_TEST_MAX_CONCURRENCY:-"-1"}
```

Comment:
ic, this is set in 

```
rep -r KUBE_INTEGRATION_TEST_MAX_CONCURRENCY hack/
hack/jenkins/test-dockerized.sh:export KUBE_INTEGRATION_TEST_MAX_CONCURRENCY=4
```

Reply:
TODO: can we auto-set this better?
but at least we have a standard way to plumb it now.